### ISSUE TITLE
Support Cycle Unified

### ISSUE THREAD

**Issue by wyqydsyq** [Widdershin/cycle-restart#61]
Currently `cycle-restart` is unpredictable when using Cycle Unified. After updating one of my projects where everything previously worked flawlessly to use Unified I noticed a few drivers not working as expected or not at all when being restarted. To be considered compatible with Cycle Unified, client-side standard drivers (DOM, HTTP, History) should all work out of the box (excluding wrapper configuration e.g. `pauseSinksWhileReplaying`). I'll be updating this task to track issues relevant to this as I encounter them. 

- [ ] https://github.com/Widdershin/cycle-restart/issues/62 DOM is alternatingly wiped and restored upon HMR updates

**Comment by Widdershin** [comment]
> I've found it thrashes the DOM driver alternating with each HMR update. So on initial load it's fine, first HMR update the DOM is nuked, second HMR update the DOM is restored and rendered to what you would expect the result of the code in that HMR update to produce, but then the next HMR update after that nukes it again and so on.

Can you please provide a sample application where this occurs?

> There's a few other issues relating to various (standard/core) drivers I don't have time to expand on right now, will follow up to create issues for those (and probably change this one to a checkbox list of tracking the others).

Sounds good. Definitely open them as individual issues.

>  I've noticed this project hasn't had much progress lately. Is Unity support something you've been working on in the shadows or are you looking for contributors? I'd be happy to submit PRs to try and help getting it working with Cycle Unity as I really want fully functional HMR with state restoration in my new boilerplate project 😄

I landed what was effectively 6 months of work to support xstream + diversity 2 months ago. Since then I haven't had much time for this project.

I definitely welcome contributions, and I would love to have a very stabled version for Cycle Unified. Most of the recent progress towards a better cycle-restart has been about @cycle/time, so that I can address issues like #51.

In short, please open issues and make pull requests. I would love your help.

**Comment by wyqydsyq** [comment]
I haven't been able to find any more issues using this with unified beyond #62 so I'll close this ticket.
